### PR TITLE
Helm template upgrade to auto-generate co-pilot configuration

### PR DESCRIPTION
**Pull request by kumare3**
Auto-generates a new cluster configuration template for copilot that has the storage configuration.

Signed-off-by: Ketan Umare <ketan.umare@gmail.com>

### CODE REVIEW

File: helm/templates/_helpers.tpl [hunk 1]
```diff
@@ -198,4 +198,46 @@ storage:
...
+{{- define "copilot.config" -}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: flyte-data-config
+  namespace: {{`{{ namespace }}`}}
+data:
+  config.yaml: |
+    storage:
+{{- if eq .Values.storage.type "s3" }}
```
Comment (EngHabu) [inline helm/templates/_helpers.tpl:212]: Can't we do `{{ include storage }}` here?

Comment (sbrunk) [reply helm/templates/_helpers.tpl:212]: Agreed if that's possible we should avoid the duplication here.

Comment (kumare3) [reply helm/templates/_helpers.tpl:212]: Hmm let me try

Comment (kumare3) [reply helm/templates/_helpers.tpl]: Done!

**Review by EngHabu — APPROVED** [review]

**Pull request merged**